packages/cli/src/eventbus/MessageEventBusWriter/MessageEventBusLogWriterWorker.ts

```diff
@@ -25,6 +25,24 @@ function setKeepFiles(keepNumberOfFiles: number) {
 	keepFiles = keepNumberOfFiles;
 }
 
+function buildRecoveryInProgressFileName(): string {
+	return `${logFileBasePath}.recoveryInProgress`;
+}
+
+function startRecoveryProcess() {
+	if (existsSync(buildRecoveryInProgressFileName())) {
+		return false;
+	}
+	openSync(buildRecoveryInProgressFileName(), 'a');
```

Comment:
Shouldn't we open and close the file, since we just want to create it?

Reply:
good point, fixed